Git Formats/Git Log.sublime-syntax

```diff
@@ -13,13 +13,33 @@ contexts:
   main:
     # 1st header line
     # commit d9d9fb804f5d61c13ba2f8746af33a9f3c609075
-    - match: (?:(commit)\s+(\h{7,}))?\s*\n
+    - match: \b(commit)\s+(\h{7,})
       scope: meta.header.git.commit markup.raw.block.git.log
       captures:
         1: keyword.other.commit.git.log
         2: constant.other.hash.git.log
-      embed: commit-header
-      escape: (?=^commit\s)
+      push:
+        - match: $\n?
+          scope: meta.header.git.commit markup.raw.block.git.log
+          embed: commit-header
+          escape: (?=^commit\s)
+          pop: true
+        - match: \s*(\()(HEAD)\s*(->)
```

Comment:
Why the leading `\s*`? It's not needed, IMHO.

Reply:
removed :)